src/libOpenImageIO/CMakeLists.txt

```diff
@@ -149,6 +153,9 @@ target_link_libraries (OpenImageIO
             ${format_plugin_libs} # Add all the target link libraries from the plugins
             $<TARGET_NAME_IF_EXISTS:OpenColorIO::OpenColorIO>
             $<TARGET_NAME_IF_EXISTS:OpenColorIO::OpenColorIOHeaders>
+            $<TARGET_NAME_IF_EXISTS:yaml-cpp::yaml-cpp>
+            $<TARGET_NAME_IF_EXISTS:pystring::pystring>
+            $<TARGET_NAME_IF_EXISTS:EXPAT::EXPAT>
```

Comment:
I think the most recent version of OCIO might fixed the issue that lef me to add this.

Reply:
Gone